scalability_and_performance/ztp-factory-install-clusters.adoc

```diff
@@ -0,0 +1,140 @@
+// Telcodocs-403 assembly
+:_content-type: ASSEMBLY
+[id="ztp-factory-workflows"]
+= Zero touch provisioning (ZTP) for factory workflows
+include::_attributes/common-attributes.adoc[]
+:context: ztp-factory-install
+
+:FeatureName: ZTP for factory workflows
+
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+toc::[]
+Zero touch provisioning (ZTP) for factory workflows accelerates the deployment of OpenShift with pre-certified hardware and software for rapid edge deployments.
+
+ZTP for factory workflows enables original equipment manufacturer (OEM) partners to pre-install OpenShift at their factory and build turnkey solutions on their hardware. This approach is well suited to a range of different industries such as healthcare, manufacturing, aerospace, to media, entertainment, retail, and telecommunications.
+
+ZTP for factory workflows installs the components that enable you to use {product-title} as a disconnected hub cluster. This hub cluster is then able to deploy edge clusters that can be shipped off site for final configuration.
+
+At the factory, the OEM partner first deploys a hub {product-title} cluster and then uses the hub cluster to deploy one or more edge clusters at scale.
+
+Different topologies are supported using the ZTP factory workflows. The hub cluster can be a single-node OpenShift cluster deploying multiple single-node OpenShift edge clusters or a compact cluster deploying 3 control plane and 1 worker node edge clusters at scale.
```

Comment:
Let's get a clarification on this.

IIUC, the hub can be either compact or SNO and edge can be any of SNO,compact, compact + day2 worker. 
@flaper87 @jparrill

Reply:
Yes I reached out to @flaper87 for this and constructed table from reply as to what was supported combination wise